windows/DateTimePickerWindows/DatePickerComponent.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "DatePickerComponent.h"
+#include "DateTimeHelpers.h"
+
+namespace winrt::DateTimePicker::Components {
+
+DatePickerComponent::DatePickerComponent() {
+  m_control = winrt::Microsoft::UI::Xaml::Controls::CalendarDatePicker{};
```

Comment:
```suggestion
DatePickerComponent::DatePickerComponent()
    : m_control{winrt::Microsoft::UI::Xaml::Controls::CalendarDatePicker{}} {
}
```

Initializer lists are better to initialize members in C++.  Refer [Should my constructors use “initialization lists” or “assignment”?](https://isocpp.org/wiki/faq/ctors#init-lists).